content/en/blog/2023-05-01-april-update/index.md

```diff
@@ -0,0 +1,245 @@
+- "The team has been working on ..."
+
+-->
+
+#### Weave GitOps
+
+#### Terraform-controller
+
+#### Flux Subsystem for Argo
+
+#### Azure GitOps
+
+#### VS Code GitOps Extension
+
+#### New additions to the Flux Ecosystem
+
+## Recent & Upcoming Events
+
+It's important to keep you up to date with new features and developments
+in Flux and provide simple ways to see our work in action and chat with
+our engineers.
+
+### Recent Events (ICYMI) 📺
+
+We feel blessed to have such a big community of users, contributors and
+integrators and so many are happy to talk about their experiences. In
+April here are a couple of talks we would like to highlight.
+
+Here is a list of additional videos and topics we really enjoyed -
+please let us know if we missed anything of interest and we will make
+sure to mention it in the next post!
+
+### Upcoming Events 📆
+
+We are happy to announce that we have a number of events coming up in
+May - tune in to learn more about Flux and GitOps best practices,
+get to know the team and join our community.
+
+#### Flux Bug Scrub
+
+Our Flux Bug Scrubs still are happening on a weekly basis and remain one
+of the best ways to get involved in Flux. They are a friendly and
+welcoming way to learn more about contributing and how Flux is organised
+as a project.
+
+The next dates are going to be:
+
+- [2023-xx-yy 12:00 UTC, 14:00 CEST](/#calendar)
+- [2023-xx-yy 17:00 UTC, 19:00 CEST](/#calendar)
+- [2023-xx-yy 12:00 UTC, 14:00 CEST](/#calendar)
+- [2023-xx-yy 17:00 UTC, 19:00 CEST](/#calendar)
+
+We are flexible with subjects and often go with the interests of the
+group or of the presenter. If you want to come and join us in either
+capacity, just show up or if you have questions, reach out to Kingdon on
+Slack.
+
+We really enjoyed this [demo of the k3d git
+server](https://www.youtube.com/watch?v=hNt3v0kk6ec)
+recently. It's a local Git server that runs outside of Kubernetes, to
+support offline dev in a realistic but also simple way that does not
+depend on GitHub or other hosted services.
+
+## In other news
+
+### Michael Fornaro joins Flux as a Project Member
+
+We are very grateful that Michael joins the team as a [project
+member](https://github.com/fluxcd/community/blob/main/community-roles.md#project-member)!
+In the past months he has been very active in Slack #flux channels with over
+400+ responses and threads. He also contributed to the Flux Bug Scrub sessions;
+Kingdon says:
+
+> We're now working together to expand the Bug Scrub, the first AEST "bug scrub"
+> session was last week. This is meant to include more time zones who would
+> typically not be able to attend (AEST-centric scheduling would be more accessible
+> to Eastern Europe, India, Southeast Asia, etc.)
+
+In addition to that Michael was also involved through being a co-founder at
+[k8s@Home](https://github.com/k8s-at-home/)
+Github & [Discord](https://discord.gg/sTMX7Vh) where they support Flux users as
+well.
+
```

Comment:
@xunholy Does this look good to you? I could also include https://avatars.githubusercontent.com/u/20387402 as your profile pic if you like?